docs/src/running-validator/validator-reqs.md

```diff
@@ -110,6 +117,5 @@ the internet on staked, mainnet-beta validators.
 
 ## GPU Requirements
```

Comment:
I don't know a single validator that uses a GPU and will could lead to additional head aches if problems arise. I would strongly consider removing the GPU part entirely.

Reply:
Removed